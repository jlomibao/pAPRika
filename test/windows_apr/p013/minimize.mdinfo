

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1128E+02     4.6897E-01     1.5987E+00     H34       107

 BOND    =        1.1100  ANGLE   =      107.7352  DIHED      =       51.4528
 VDWAALS =      -46.2816  EEL     =     1337.5492  EGB        =     -135.8571
 1-4 VDW =        5.2113  1-4 EEL =    -2132.2112  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================
