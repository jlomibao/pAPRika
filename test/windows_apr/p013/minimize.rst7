defa
  125
  -0.5217212   4.9759776   1.8352362   1.0157700   4.9108517   1.8351772
  -0.8854978   4.2791024   0.6084110  -0.8854113   4.2789195   3.0619813
  -0.9016903   6.0041154   1.8353254   1.4813290   5.9031716   1.8352344
   1.3195573   4.1855767   0.6085969   1.3196403   4.1853829   3.0616233
   2.6868600   4.0833360   0.1137295   2.6869699   4.0831189   3.5564023
   0.2013696   3.8584375   3.7829819   0.2012355   3.8587711  -0.1127434
   0.1802851   3.3487280   4.8903099   0.1800723   3.3492864  -1.2201651
   2.6633077   4.0469778  -0.9815179   3.2459808   4.9656360   0.4459460
   2.6634694   4.0466042   4.6516456   3.2460574   4.9654770   3.2242836
  -4.6359619   1.9609800   1.8353110  -3.8101232   3.2595418   1.8352708
  -4.2146245   1.2971716   0.6085621  -4.2146231   1.2971707   3.0620729
  -5.7162450   2.1465396   1.8353156  -4.4362892   4.1591862   1.8352879
  -3.0303185   3.1594966   0.6084925  -3.0303129   3.1594792   3.0620825
  -2.2565864   4.2914468   0.1135941  -2.2564416   4.2913483   3.5569486
  -3.3067311   2.0275210   3.7831765  -3.3067562   2.0275320  -0.1125719
  -2.8757689   1.7539242   4.8903612  -2.8758539   1.7539554  -1.2197741
  -2.2363537   4.2526050  -0.9816344  -2.7399999   5.2175097   0.4454920
  -2.2360720   4.2524520   4.6521726  -2.7398256   5.2174653   3.2251589
  -4.0834516  -3.1098993   1.8352133  -4.7950330  -1.7453607   1.8352663
  -3.2979793  -3.0769665   0.6084092  -3.2978085  -3.0769723   3.0619110
  -4.7843424  -3.9526305   1.8352568  -5.8872402  -1.8377042   1.8353112
  -4.3184488  -1.1200751   0.6084675  -4.3183295  -1.1200999   3.0620325
  -4.9114301   0.1162750   0.1137864  -4.9113365   0.1161990   3.5568233
  -3.4762224  -1.9255310   3.7830430  -3.4764628  -1.9255100  -0.1126712
  -3.0236610  -1.6891395   4.8901765  -3.0240584  -1.6891123  -1.2198601
  -4.8678187   0.1144702  -0.9814479  -5.9550597   0.1610912   0.4458497
  -4.8676670   0.1143174   4.6520552  -5.9549901   0.1609832   3.2248311
   0.5850529  -5.1654699   1.8348694  -0.9524992  -5.1002279   1.8349741
   0.9488674  -4.4684607   0.6081071   0.9490176  -4.4685696   3.0616486
   0.9649583  -6.1936312   1.8347928  -1.4181237  -6.0925170   1.8349706
  -1.2561380  -4.3749235   0.6082451  -1.2559799  -4.3749870   3.0617900
  -2.6234454  -4.2707797   0.1137480  -2.6232230  -4.2708041   3.5564661
  -0.1375609  -4.0479407   3.7828207  -0.1378127  -4.0478196  -0.1129059
  -0.1162307  -3.5378841   4.8899902  -0.1166308  -3.5377300  -1.2200564
  -2.6002525  -4.2341319  -0.9814974  -3.1837668  -5.1522725   0.4460846
  -2.5998774  -4.2341554   4.6517083  -3.1836118  -5.1522848   3.2242110
   4.6985579  -2.1491066   1.8347173   3.8734227  -3.4481209   1.8347225
   4.2767977  -1.4854400   0.6080317   4.2768976  -1.4855073   3.0614706
   5.7789389  -2.3340955   1.8346663   4.5000371  -4.3474531   1.8346760
   3.0934875  -3.3484127   0.6079884   3.0935854  -3.3484663   3.0615240
   2.3199577  -4.4805933   0.1132849   2.3201744  -4.4807048   3.5562668
   3.3694340  -2.2163599   3.7826108   3.3693118  -2.2162897  -0.1130827
   2.9384189  -1.9430111   4.8898347   2.9382643  -1.9429278  -1.2202840
   2.2996667  -4.4418813  -0.9819471   2.8035461  -5.4065295   0.4452816
   2.3000481  -4.4420760   4.6515048   2.8037389  -5.4065993   3.2241193
   4.1488007   2.9227387   1.8349372   4.8584236   1.5570412   1.8348289
   3.3640849   2.8906207   0.6075378   3.3641860   2.8904835   3.0624036
   4.8510157   3.7643669   1.8349502   5.9507647   1.6477865   1.8347707
   4.3808215   0.9320375   0.6082253   4.3809494   0.9319323   3.0614344
   4.9732598  -0.3044141   0.1130458   4.9734436  -0.3045433   3.5564596
   3.5393781   1.7378953   3.7824502   3.5392278   1.7381079  -0.1126499
   3.0853013   1.5010906   4.8888751   3.0851097   1.5014476  -1.2190810
   4.9290290  -0.3024360  -0.9821634   6.0171027  -0.3493612   0.4444202
   4.9293449  -0.3026415   4.6516743   6.0172430  -0.3494784   3.2249466
   0.0001525   0.0031276  13.7198083  -0.9139691  -0.5758013  13.5397634
   0.8629919  -0.6497633  13.5397124   0.0341710   0.8210077  12.9896604
   0.0234034   0.5608988  15.1560117   0.9308217   1.1613217  15.2948665
  -0.0231990  -0.5608197  16.2153819  -0.8310259   1.2345812  15.2949317
  -0.9305029  -1.1612410  16.0757859   0.8311945  -1.2344095  16.0758065
  -0.0000933  -0.0040310  17.6522385  -0.8630017   0.6488187  17.8321581
  -0.0341385  -0.8224958  18.3817229   0.9140106   0.5749473  17.8322083
  -0.0001102   0.0002576  -6.0041010  -0.0002036   0.0000541  -9.0000704
   0.0001672   2.2000035 -11.2000237
