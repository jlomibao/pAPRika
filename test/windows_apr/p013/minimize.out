
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:35:16

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p013
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   569510 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9697E+02     1.0074E+01     2.4799E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.7093  EEL     =     1326.5914  EGB        =     -129.5523
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9680
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1128E+02     4.6905E-01     1.5986E+00     H34       107

 BOND    =        1.1083  ANGLE   =      107.7344  DIHED      =       51.4550
 VDWAALS =      -46.2772  EEL     =     1337.4975  EGB        =     -135.8441
 1-4 VDW =        5.2090  1-4 EEL =    -2132.1742  RESTRAINT  =        0.0097
 EAMBER  =     -811.2915
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1128E+02     4.6905E-01     1.5986E+00     H34       107

 BOND    =        1.1084  ANGLE   =      107.7345  DIHED      =       51.4547
 VDWAALS =      -46.2777  EEL     =     1337.5029  EGB        =     -135.8455
 1-4 VDW =        5.2093  1-4 EEL =    -2132.1780  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1128E+02     4.6904E-01     1.5986E+00     H34       107

 BOND    =        1.1086  ANGLE   =      107.7345  DIHED      =       51.4545
 VDWAALS =      -46.2782  EEL     =     1337.5089  EGB        =     -135.8470
 1-4 VDW =        5.2095  1-4 EEL =    -2132.1823  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1128E+02     4.6903E-01     1.5986E+00     H34       107

 BOND    =        1.1088  ANGLE   =      107.7346  DIHED      =       51.4542
 VDWAALS =      -46.2787  EEL     =     1337.5147  EGB        =     -135.8485
 1-4 VDW =        5.2098  1-4 EEL =    -2132.1864  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1128E+02     4.6902E-01     1.5986E+00     H34       107

 BOND    =        1.1090  ANGLE   =      107.7348  DIHED      =       51.4540
 VDWAALS =      -46.2792  EEL     =     1337.5206  EGB        =     -135.8499
 1-4 VDW =        5.2100  1-4 EEL =    -2132.1907  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1128E+02     4.6901E-01     1.5986E+00     H34       107

 BOND    =        1.1092  ANGLE   =      107.7348  DIHED      =       51.4538
 VDWAALS =      -46.2797  EEL     =     1337.5261  EGB        =     -135.8514
 1-4 VDW =        5.2103  1-4 EEL =    -2132.1945  RESTRAINT  =        0.0097
 EAMBER  =     -811.2913
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1128E+02     4.6900E-01     1.5987E+00     H34       107

 BOND    =        1.1094  ANGLE   =      107.7350  DIHED      =       51.4535
 VDWAALS =      -46.2801  EEL     =     1337.5320  EGB        =     -135.8528
 1-4 VDW =        5.2105  1-4 EEL =    -2132.1989  RESTRAINT  =        0.0097
 EAMBER  =     -811.2915
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1128E+02     4.6899E-01     1.5987E+00     H34       107

 BOND    =        1.1096  ANGLE   =      107.7350  DIHED      =       51.4533
 VDWAALS =      -46.2806  EEL     =     1337.5372  EGB        =     -135.8542
 1-4 VDW =        5.2108  1-4 EEL =    -2132.2025  RESTRAINT  =        0.0097
 EAMBER  =     -811.2913
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1128E+02     4.6898E-01     1.5987E+00     H34       107

 BOND    =        1.1098  ANGLE   =      107.7351  DIHED      =       51.4530
 VDWAALS =      -46.2811  EEL     =     1337.5431  EGB        =     -135.8556
 1-4 VDW =        5.2111  1-4 EEL =    -2132.2067  RESTRAINT  =        0.0097
 EAMBER  =     -811.2913
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1128E+02     4.6897E-01     1.5987E+00     H34       107

 BOND    =        1.1100  ANGLE   =      107.7352  DIHED      =       51.4528
 VDWAALS =      -46.2816  EEL     =     1337.5492  EGB        =     -135.8571
 1-4 VDW =        5.2113  1-4 EEL =    -2132.2112  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1128E+02     4.6897E-01     1.5987E+00     H34       107

 BOND    =        1.1100  ANGLE   =      107.7352  DIHED      =       51.4528
 VDWAALS =      -46.2816  EEL     =     1337.5492  EGB        =     -135.8571
 1-4 VDW =        5.2113  1-4 EEL =    -2132.2112  RESTRAINT  =        0.0097
 EAMBER  =     -811.2914
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.306    9.302    0.004    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.399  147.398    0.001    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.011   62.003    0.009    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.193 -146.178    0.015    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.551  -72.548    0.003    0.000 t
  DUM  DM1    3 --  C    BUT    2:   19.724   19.765    0.041    0.008 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.995  180.000    0.005    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.887  180.000    0.113    0.000 a
                                       Total distance penalty:      0.008
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.24 (21.48% of Gen B)
|                Calc gb diag               4.07 (38.93% of Gen B)
|                Calc gb off-diag           4.13 (39.56% of Gen B)
|                Other                      0.00 ( 0.03% of Gen B)
|             Gen Born time             10.45 (100.0% of Nonbo)
|          Nonbond force             10.45 (90.34% of Force)
|          Bond/Angle/Dihedral        1.08 ( 9.35% of Force)
|          Other                      0.04 ( 0.31% of Force)
|       Force time                11.57 (100.0% of Runmd)
|    Runmd Time                11.57 (99.81% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.59 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:35:16.566  on 04/10/2019
|           Setup done at 18:35:16.571  on 04/10/2019
|           Run   done at 18:35:28.154  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
